snapcraft/cli/help.py

```diff
@@ -65,8 +75,16 @@ def help_command(topic, devel):
         try:
             _module_help(topic, devel)
         except ImportError:
-            echo.error('The plugin does not exist. Run `snapcraft '
-                       'list-plugins` to see the available plugins.')
+            click.echo(dedent("""\
+    The argument to help is not a plugin name nor topic, to see the
+    available topics run:
+
+        snapcraft help topics
+
+    to see the available plugins run:
+
+        snapcraft list-plugins
+    """))
```

Comment:
To make this message consistent with the others on the errors pr branch, I would change it to something like:
```
Failed to display help: '{topic}' is not a plugin name nor a topic. To see the available topics, run `snapcraft help topics`. To see the available plugins, run `snapcraft list-plugins`.
```

Reply:
That has no wrapping, doesn't seem elegant to me.

Reply:
In my proposal, I left the messages as short as possible, with no unnecessary white spaces. And left the wrapping to the terminal. This was not suggested by mpt, just my suggestion. I don't have a strong opinion on the format, just that it should be consistent, so I can adjust my pr if you prefer a different style.

Reply:
The wrapping on the terminal will cut words in half, the only place where we shouldn't cut is for links so they can still open.

If you look at the in-master errors, some have `\n` for exactly this reason.

Reply:
Also, consider this the help for running a command incorrectly, not necessarily an error in "snapcraft" operation.

Reply:
I agree with @elopio that it’s best to repeat the unknown command, since that would help you realize if you’d mistyped it. (But ellipsize it at, say, 64 characters, in case you accidentally pasted a whole StackOverflow page into the terminal.)

However, in this case I don’t actually think it’s a good idea to follow the “what went wrong, why it happened, what you can do” format, for the same reason as I don’t think it’s a good idea to use the word “run”: looking up help doesn’t feel like a command. Enough just to begin ‘There is no help topic or plugin “fqwhgads”. Try:’

> The wrapping on the terminal will cut words in half, the only place where we shouldn't cut is for links so they can still open.

When output is mostly words (like it is for help, and for error messages), should it perhaps go through a formatter that word-wraps it to the current terminal width? A sprinkling of `\n` will sometimes be too much and sometimes not enough.

Reply:
I've added a formatter, the same one we use for help already.